src/types/SortType.ts

```diff
@@ -0,0 +1,5 @@
+export enum SortType {
+  DEFAULT,
+  ALPHABETIBALLY,
```

Comment:
There seems to be a typo in the enum value `ALPHABETIBALLY`. It should be corrected to `ALPHABETICALLY` to avoid confusion and potential bugs when using this enum.